tests/pos-with-compiler-cc/dotc/core/SymbolLoaders.scala

```diff
@@ -171,7 +174,9 @@ object SymbolLoaders {
           Nil)
       }
 
-      val unit = CompilationUnit(ctx.getSource(src))
+      val unit = inDetachedContext:
+        CompilationUnit(ctx.getSource(src, scala.io.Codec(ctx.settings.encoding.value)))
+            // !cc! default arguments can't be handled
```

Comment:
minor: does this mean that there are problems when CC interacts with default parameters?

Reply:
And if so, shall we open an issue to track and solve it?